### DISCUSSION TITLE
Can not find User Picker Field in version 1.0.0-rc2-13450

### DISCUSSION THREAD

**Discussion by EricBui619** [OrchardCMS/OrchardCore#9038] [Q&A] [answered]
Anyone can explain to me why  User Picker Field doesn't appear in my Orchard Core ? My current version is 1.0.0-rc2-13450 from NuGet, also the latest stable version in master. I checked out the latest version rc2 released recently in develop and could seen User Picker Field in UI. So how can I use User Picker Field? Or I have to wait new release version to be merged to master?

![image](https://user-images.githubusercontent.com/30250345/113389140-b469d680-93b9-11eb-87ac-a493af8140f2.png)

**Comment by deanmarcussen** [comment, marked as answer]
The UserPickerField is not available in RC2.

You will either have to wait for a new release or upgrade to the dev feed. 
https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/

**Reply by EricBui619** [reply]
Thank you for your suggestion
